ghost/core/core/server/data/schema/schema.js

```diff
@@ -600,6 +600,45 @@ module.exports = {
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
     },
+    subscriptions: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        type: {
+            type: 'string', maxlength: 50, nullable: false, validations: {
+                isIn: [['free', 'comped', 'paid']]
+            }
+        },
+        status: {
+            type: 'string', maxlength: 50, nullable: false, validations: {
+                isIn: [['active', 'expired', 'cancelled']]
+            }
+        },
+        member_id: {type: 'string', maxlength: 24, nullable: false, unique: false, references: 'members.id', cascadeDelete: true},
+        tier_id: {type: 'string', maxlength: 24, nullable: true, unique: false, references: 'products.id'},
+
+        // These are null if type !== 'paid'
+        cadence: {
+            type: 'string', maxlength: 50, nullable: true, validations: {
+                isIn: [['month', 'year']]
+            }
+        },
+        currency: {type: 'string', maxLength: 3, nullable: true},
+        amount: {type: 'integer', nullable: true},
+        
+        // e.g. 'stripe'
```

Comment:
I copied these example values from the spec and left them as I liked the idea of having examples right next to the column definition. How do you @daniellockyer @allouis feel about having slightly more information next to the schema definition?

Reply:
Fine by me!

Reply:
Me too 🙂